<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
    <item row="6" column="0">
     <widget class="QPushButton" name="AddCity">
      <property name="text">
       <string>Add City</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_test">
      <property name="placeholderText">
       <string>Ihre Eingabe</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="tesDrawtMapButton">
      <property name="text">
       <string>Test Map</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="testAbstractMap">
      <property name="text">
       <string>Test Abstract Map</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="testDrawCityButton">
      <property name="text">
       <string>Test Draw City</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QPushButton" name="testDrawStreetButton">
      <property name="text">
       <string>Test Street</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QPushButton" name="testDijkstra">
      <property name="text">
       <string>Test Dijkstra</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="fillMap">
      <property name="text">
       <string>Fill Map</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="addMotorWay">
      <property name="text">
       <string>Add Motorway</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_test">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;enter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>pushButton_test</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_eingabe">
      <property name="text">
       <string>Ihre Eingabe</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="AddStreet">
      <property name="text">
       <string>Add Street</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="checkBoxTest">
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="20">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="addStateRoad">
      <property name="text">
       <string>Add State Road</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionClear_Scene"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionTrennerhinzufuegen">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
